src/main/java/org/terasology/climateConditions/ClimateConditionsSystem.java

```diff
@@ -57,28 +64,30 @@ public void addHumidityModifier(float order, ConditionModifier humidityModifier)
         humidityModifiers.put(order, humidityModifier);
     }
 
-    public void configureTemperature(int seaLevel, int maxLevel, float diversity, Function<Float, Float> function,
-                                     float minimumValue, float maximumValue) {
-        int seed = worldSeed.hashCode();
+    public void configureTemperature() {
+        configureTemperature(32, 10, -10, 10);
```

Comment:
Magic numbers? Might be worth to extract to meaningfully named constants holding default values (beyond just checking the variable names in `configureTemperature`)